x-pack/plugins/security_solution/public/timelines/components/flyout/pane/index.tsx

```diff
@@ -38,6 +38,18 @@ const FlyoutPaneComponent: React.FC<FlyoutPaneComponentProps> = ({
     [timelineId]
   );
 
+  const handleResize = useCallback(() => {
+    setFlyoutPaneWidth(ref.current?.offsetWidth);
+  }, []);
+
+  useEffect(() => {
+    window.addEventListener('resize', handleResize);
```

Comment:
That seems like a workaround. Can we add the comment why this logic was added?

Reply:
The current styling looks like this on serverless version:

<img width="2553" alt="Screenshot 2023-06-21 at 11 32 29" src="https://github.com/elastic/kibana/assets/6295984/11e7a722-8a81-440e-bb31-66ab873020ce">

The timeline header is cropped and the side nav occupies the left side when timeline is opened.
Adding the resize handler was hopefully to set the right width for the timeline content when navigation open/close and screen resize.

But it seems not working fine.


https://github.com/elastic/kibana/assets/6295984/8c13f160-3e36-4b76-aec4-dcd48fd25651

I'll revert this change and write up an issue for timeline's styling and take it from there.